src/pages/Models/Details/ModelDetails.js

```diff
@@ -145,7 +145,14 @@ const generateUnitRows = modelStatusData => {
             content: unit.publicAddress.split(":")[-1] || "-",
             className: "u-align--right"
           },
-          { content: unit.workloadStatus.info }
+          {
+            content: (
```

Comment:
This change is a bit odd as the truncation ellipsis spills out of the table.

<img width="399" alt="Screen Shot 2020-01-22 at 11 06 44 AM" src="https://user-images.githubusercontent.com/532033/72916237-4a5e9780-3d07-11ea-9da3-ed337cd2c4c6.png">